gcloud/storage/blob.py

```diff
@@ -276,17 +280,34 @@ def delete(self, client=None):
         """
         return self.bucket.delete_blob(self.name, client=client)
 
-    def download_to_file(self, file_obj, client=None):
+    def download_to_file(self, file_obj, key=None, client=None):
         """Download the contents of this blob into a file-like object.
 
         .. note::
 
            If the server-set property, :attr:`media_link`, is not yet
            initialized, makes an additional API request to load it.
 
+         Downloading a file that has been `customer-supplied
+         <https://cloud.google.com/storage/docs/encryption#customer-supplied>`_
+         encryption::
+
+            >>> from gcloud import storage
+            >>> from gcloud.storage import Blob
+
+            >>> sc = storage.Client(project='my-project')
```

Comment:
What is the variable name we use elsewhere? Let's try to be consistent. AFAIK it's `client`, rather than `sc`